maat-court-data-api/src/main/java/gov/uk/courtdata/entity/FdcContributionsEntity.java

```diff
@@ -0,0 +1,98 @@
+package gov.uk.courtdata.entity;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import gov.uk.courtdata.enums.FdcContributionsStatus;
+import jakarta.persistence.*;
+import jakarta.xml.bind.annotation.XmlType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "FDC_CONTRIBUTIONS", schema = "TOGDATA")
+@XmlType
+public class FdcContributionsEntity {
+
+    @Id
+    @SequenceGenerator(name = "fdc_contributions_gen_seq", sequenceName = "S_GENERAL_SEQUENCE", allocationSize = 1, schema = "TOGDATA")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "fdc_contributions_gen_seq")
+    @Column(name = "ID", nullable = false)
+    private Integer id;
+
+    @Column(name= "CONT_FILE_ID", nullable = false)
```

Comment:
It looks like `CONT_FILE_ID` is nullable in the database, should it be marked `nullable = false` here? Or maybe the DB should have this constraint added?

Reply:
It being nullable is the correct course of action.